platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/ClassPageRenderer.java

```diff
@@ -120,12 +120,7 @@ protected void renderIgnoredTests(SimpleHtmlWriter htmlWriter) throws IOExceptio
                 .startElement("a").attribute("name", test.getId().toString()).characters("").endElement() //browsers dont understand <a name="..."/>
                 .startElement("h3").attribute("class", test.getStatusClass()).characters(test.getDisplayName()).endElement();
             for (SerializableFailure failure : test.getFailures()) {
-                String message;
-                if (GUtil.isTrue(failure.getMessage()) && !failure.getStackTrace().contains(failure.getMessage())) {
-                    message = failure.getMessage() + SystemProperties.getInstance().getLineSeparator() + SystemProperties.getInstance().getLineSeparator() + failure.getStackTrace();
-                } else {
-                    message = failure.getStackTrace();
-                }
+                String message = failure.getMessage() + SystemProperties.getInstance().getLineSeparator() + SystemProperties.getInstance().getLineSeparator() + failure.getStackTrace();
```

Comment:
Minor nit, but can this line separator call be pulled out to a variable outside the loop? Seems wasteful to refetch it over and over.